config/config.go

```diff
@@ -58,7 +58,7 @@ func get() (*Config, error) {
 		IsPublishingMode:           false,
 		Languages:                  "en,cy",
 		SiteDomain:                 "localhost",
-		CensusFirstResults:         "Wednesday, 27-Jul-22 11:00:00 BST", // Use BST as timezone
+		CensusFirstResults:         "Wednesday, 27-Jul-23 11:00:00 BST", // Use BST as timezone
```

Comment:
If the CheckTime function is failing because of the default configuration value then we should not use the configuration in our unit tests in CheckTime.

I have fixed this in this PR - https://github.com/ONSdigital/dp-frontend-homepage-controller/pull/124/files#diff-d1553f2592e03735236f765108820b5a205183970f19befe0d58805e89b2ada8

If you can review this one, then I can merge it into develop and you can merge it down into your branch. And revert changes to default value